vg/len.go

```diff
@@ -29,3 +35,36 @@ func (l Length) Dots(dpi float64) float64 {
 func (l Length) Points() float64 {
 	return float64(l)
 }
+
+func (l *Length) String() string {
+	return fmt.Sprintf("%vpt", float64(*l))
+}
+
+// Set is the method to set a flag value, part of the flag.Value interface.
+// e.g. "42cm" "2.4inch" "66pt"
```

Comment:
The comment should enumerate all available abbreviations, so that no one has to hunt through the code for them.